charts/aws-pca-issuer/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   repository: public.ecr.aws/k1n1h4h4/cert-manager-aws-privateca-issuer
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "latest"
+  # tag: "latest"
```

Comment:
We tested the change and were able to pull the correct image tag for each version, so this change is working as expected! Just a nitpick for style's sake; could you remove the line entirely instead of just commenting, as well as the comment on line 8? Thanks!